zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraSpanStore.scala

```diff
@@ -35,6 +35,7 @@ object CassandraSpanStoreDefaults {
   val SpanTtl = 7.days
   val IndexTtl = 3.days
   val MaxTraceCols = 100000
+  val MaxConnections = 2
```

Comment:
I guess driver threats cassandra installation as non-local, due to non-default dc name we use. Hope this will be changed after #967